modules/knative-serving-concurrent-autoscaling-requests.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * serverless/configuring-knative-serving-autoscaling.adoc
+
+[id="knative-serving-concurrent-autoscaling-requests_{context}"]
+= Configuring concurrent requests for Knative Serving autoscaling
+
+You can specify the number of concurrent requests that should be handled by each instance of an application (revision container) by adding the `target` annotation in the revision template, or by modifying the value for `containerConcurrency` in the Knative Serving ConfigMap.
+
+NOTE: `containerConcurrency` should only be used if there is a clear need to limit how many requests reach the application at a given time. Using `containerConcurrency` is only advised if the application needs to have an enforced constraint of concurrency.
+
+== Configuring concurrent requests using revision templates
+
+The default limit for the number of concurrent requests is `100`, which is defined in the Knative Serving ConfigMap, but you can override this value by adding or modifying the `autoscaling.knative.dev/target` annotation value in the revision template.
+
+Here is an example of how this annotation would be used in the revision template to set the target to `50`.
+
+[source,yaml]
+----
+autoscaling.knative.dev/target: 50
+----
+
+== Configuring concurrent requests using the ConfigMap
```

Comment:
This heading doesn't seem to fit the following section. containerConcurrency is still part of the revision template.